app/javascript/components/admin/UserModal.js

```diff
@@ -110,19 +137,47 @@ class UserModal extends React.Component {
             </Form.Group>
             {this.props.type === 'edit' && (
               <Form.Group>
-                <Form.Label>Status</Form.Label>
-                <Form.Check
-                  id="status-input"
+                <Form.Label className="input-label">System Access</Form.Label>
+                <div style={{ display: 'flex', alignItems: 'center' }}>
+                  <Form.Check
+                    id="system-access-input"
+                    name="system-access"
+                    type="switch"
+                    checked={this.state.isLocked}
+                    label={this.state.isLocked ? 'Locked' : 'Unlocked'}
+                    onChange={this.handleLockedSystemAccessChange}
+                  />
+                  {this.state.lockReason === 'Auto-locked by the System' && this.state.isLocked && (
+                    <div className="locked-warning-text ml-2">
+                      <FontAwesomeIcon className="mr-1" icon={faExclamationCircle} />
+                      <span>{this.state.autoLockReason}</span>
+                    </div>
+                  )}
+                </div>
+              </Form.Group>
+            )}
+            {this.props.type === 'edit' && (
+              <Form.Group>
+                <Form.Label className="input-label" htmlFor="status-select">
+                  Status
+                </Form.Label>
+                <Select
+                  inputId="status-select"
+                  id="status"
                   name="status"
-                  type="switch"
-                  checked={this.state.isLocked}
-                  label={this.state.isLocked ? 'Locked' : 'Unlocked'}
-                  onChange={this.handleLockedStatusChange}
+                  value={this.getStatusValue()}
+                  options={this.state.lockReasonOptions.map(lockReason => {
```

Comment:
It says in the PR description that BLANK is a valid option for status when locked.  Is there a reason it's not included in the dropdown?  It makes sense that `Auto-locked by the System` is not included here

<img width="558" alt="Screen Shot 2022-01-07 at 5 19 19 PM" src="https://user-images.githubusercontent.com/35042815/148614785-4b91ec7e-9244-4c5b-b0aa-1b30bfd3c265.png">

Reply:
I was going based off of this part of the Story (see below), but can follow up with @hackrm to make sure of the specifics.

Allow the user to select from the following drop down options:

- "Not specified" (default section when the user is initially locked and the Active/Inactive status is removed
If selected, should be displayed as a blank cell in the Admin table and Export
- "No longer an employee"
- "No longer needs access"
- "Other"

Reply:
I was going off of this part of the story (see below), but can definitely follow up with @hackrm to clarify!

Allow the user to select from the following drop down options:

- "Not specified" (default section when the user is initially locked and the Active/Inactive status is removed
If selected, should be displayed as a blank cell in the Admin table and Export
- "No longer an employee"
- "No longer needs access"
- "Other"

Reply:
Hmm, this may have been me getting ahead of myself. When I drafted this, I believe I was thinking we'd complete this [ticket](https://tracker.codev.mitre.org/browse/SARAALERT-1076) earlier, but it was never considered because of other priorities. As a result, in the system there is no instance where the <blank> value is designated with "Not Specified." Given that <blanks> are not yet treated like this in the system, it's probably best to leave it as an acutal blank value instead of "Not Specified." Sorry for the switch up @deastman , let me know if this ends up being a bigger change for you to make than I hope it is. 

I will update the ticket description to reflect this new end goal.

Reply:
@sgober See [dab98fbc9dbfa8aea836595473daf0eb3d2aa91c](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/dab98fbc9dbfa8aea836595473daf0eb3d2aa91c)